package.json

```diff
@@ -4,7 +4,7 @@
   "description": "docker hub webhook",
   "main": "index.js",
   "scripts": {
-    "start": "micro",
+    "start": "micro -l tcp://${npm_config_HOST}:${npm_config_PORT}",
```

Comment:
this seems working only for linux machines (micro [docs](https://www.npmjs.com/package/micro)), but from the `README.md` instructions I thought the project was mainly meant for linux, is that correct ?